### ISSUE TITLE
The issue #5235   is back

### ISSUE THREAD

**Issue by The-Dark-Man** [nvaccess/nvda#6039]
NVDA 2016.2 is unable to speak grouped numbers again. In NVDA 2016.1 it worked correctly. This problem is in German language, only.

**Comment by jcsteh** [comment]
@The-Dark-Man, @bdorer: Is this now fixed in NVDA master (as per the [last comment on #5235](https://github.com/nvaccess/nvda/issues/5235#issuecomment-224324258))? Can this be closed?

**Comment by jcsteh** [comment]
I believe this to be fixed as above. Please reopen if not.
